$OpenBSD: patch-Makefile,v 1.3 2008/08/28 19:55:20 kili Exp $

Fail on failure, and don't be so silent during the tests, it makes
me nervous.

--- Makefile.orig	Fri Nov 23 16:20:30 2007
+++ Makefile	Fri Nov 23 21:14:56 2007
@@ -190,8 +190,7 @@ config: script/errnogen.c
 install:
 	./configure --install
 test:
-	cd tests && ./runtests clean >/dev/null && ./runtests >t.log 2>t.err
-	tail -4 tests/t.err
+	cd tests && ./runtests clean >/dev/null && ./runtests
 
 basic-nhc: $(PRAGMA) runtime hmake-nhc greencard-nhc compiler-nhc cpphs \
 						prelude hsc2hs libraries
